scripts/index.js

```diff
@@ -1,45 +1,54 @@
+/* eslint-disable comma-dangle */
 /* eslint-disable no-use-before-define */
 /* eslint-disable object-shorthand */
 /* eslint-disable no-undef */
+const booksLocalStorage = JSON.parse(localStorage.getItem('books') || '[]');
 const form = document.getElementById('add-book');
 let removeBtns;
+class allBooks {
+  static books = booksLocalStorage;
 
-function addBook(title, author, bookId) {
-  books.push({
-    title: title,
-    author: author,
-    id: bookId,
-  });
-  displayBooks();
-  assignRemoveEventListener();
+  static addBook(title, author, bookId) {
+    this.books.push({
+      title: title,
+      author: author,
+      id: bookId,
+    });
+    displayBooks();
+    assignRemoveEventListener();
+  }
+
+  static removeBook(removeBtn) {
+    this.books = this.books.filter(
+      (book) => book.id !== parseInt(removeBtn.id, 10)
+    );
+    localStorage.setItem('books', JSON.stringify(this.books));
+    displayBooks();
+    assignRemoveEventListener();
+  }
 }
 
 form.onsubmit = () => {
   const title = form.elements.title.value;
   const author = form.elements.author.value;
-  if (books.length) {
-    const bookId = books[books.length - 1].id + 1;
-    addBook(title, author, bookId);
+  if (allBooks.books.length) {
```

Comment:
- Kindly fix this error:
![Screen Shot 2021-10-19 at 18 36 49](https://user-images.githubusercontent.com/22258274/138004856-c5ef7647-6404-42be-be9c-4dcb556e8584.png)